docs/cloud/dev/spryker-cloud-commerce-os/add-variables.md

```diff
@@ -0,0 +1,180 @@
+
+* `type`: defines the type of a variable. Possible values:
+    * `config`: parameter
+    * `secret`: secret
+
+* `bucket`: defines what services a variable is used for. Possible values:
+    * `common`: used by all the buckets.
+    * `app`: used only by application services.
+    * `scheduler`: used by the scheduler.
+
+* `grant`: Defines access permissions to variables. Possible values:
+    * `public`: readable and writable
+    * `limited`: readable
+    * `internal`: hidden
+
+Path examples:
+
+* /fashion_club_store/staging/config/common/limited/composer_pass
+
+* /deans_jeans/prod/config/app/public/mail_host
+
+
+## Variable path hierarchy
+
+Path hierarchy is needed to cover the cases when several variables with the same name are declared. If several variables with the same name are declared, the variable with a higher priority applies. The following rules define the priority of variables:
+
+1. For any `type` and `bucket`, the priority is `public` > `limited` > `internal`.
+2. Foy any `bucket`, the priority is `bucket` > `common`.
+3. For any variable with the same name, the priority is `secret` > `config`.
+
+The following variables are arranged from lower to higher priority:
+
+* /{project}/{environment}/config/common/internal/{variable_name}
+
+* /{project}/{environment}/config/common/limited/{variable_name}
+
+* /{project}/{environment}/config/common/public/{variable_name}
+
+* /{project}/{environment}/config/{app | scheduler}/internal/{variable_name}
+
+* /{project}/{environment}/config/{app | scheduler}/limited/{variable_name}
+
+* /{project}/{environment}/config/{app | scheduler}/public/{variable_name}
+
+* /{project}/{environment}/secret/common/internal/{variable_name}
+
+* /{project}/{environment}/secret/common/limited/{variable_name}
+
+* /{project}/{environment}/secret/common/public/{variable_name}
+
+* /{project}/{environment}/secret/{app | scheduler}/internal/{variable_name}
+
+* /{project}/{environment}/secret/{app | scheduler}/limited/{variable_name}
+
+* /{project}/{environment}/secret/{app | scheduler}/public/{variable_name}
+
+## Add variables
+
+The following sections describe how to add parameters and secrets for different resources.
+
+{% info_block warningBox "Propagation of variables" %}
+
+To make variables available in your Jenkins instance, we need to terraform your added or changed variables. To do this, create a [support case](https://docs.spryker.com/docs/scos/user/intro-to-spryker/support/how-to-use-the-support-portal.html#plattform-change-requests).
+
+{% endinfo_block %}
+
+### Add parameters to all resource types
+
+1. In the AWS Management Console, go to **Services > Parameter Store**.
+2. In the **My parameters** pane, click **Create parameter**.
+    This opens the **Create parameter** page.
+3. For **Name**, enter `/{project}/{environment}/config/common/public/{variable_name}`.
+    Make sure to replace the placeholders based on your requirements.
+4. Optional: Enter a **Description**.
```

Comment:
According to google style guide, this should be rephrased to something like (see https://developers.google.com/style/ui-elements#refer-to-ui-elements):
```suggestion
4. Optional: For **Description**, enter the description of the variable.
```